examples/gallery/images/track_sampling.py

```diff
@@ -23,19 +23,45 @@
 # points
 track = pygmt.grdtrack(points=points, grid=grid, newcolname="bathymetry")
 
+# Set up colormap for Earth relief grid
+pygmt.makecpt(
+    cmap="gray",
+    output="cpt_gray_relief.cpt",
+    series=[int(grid.min()), int(grid.max()), 10],
+)
+# Set up colormap for data points of track
+pygmt.makecpt(
+    cmap="terra",
+    output="cpt_terra_points.cpt",
+    series="-1/1/0.01",  # for normalized values
+)
+
 fig = pygmt.Figure()
-# Plot the earth relief grid on Cylindrical Stereographic projection, masking
+# Plot the Earth relief grid on Cylindrical Stereographic projection, masking
 # land areas
 fig.basemap(region="g", projection="Cyl_stere/150/-20/15c", frame=True)
-fig.grdimage(grid=grid, cmap="gray")
+fig.grdimage(grid=grid, cmap="cpt_gray_relief.cpt")
 fig.coast(land="#666666")
-# Plot the sampled bathymetry points using circles (c) of 0.15 cm size
-# Points are colored using elevation values (normalized for visual purposes)
+# Plot the sampled bathymetry points using circles (c) with a diameter of
+# 0.15 centimeters (c). Points are colored using elevation values (normalized
+# for visual purposes)
 fig.plot(
     x=track.longitude,
     y=track.latitude,
     style="c0.15c",
-    cmap="terra",
+    cmap="cpt_terra_points.cpt",
     fill=(track.bathymetry - track.bathymetry.mean()) / track.bathymetry.std(),
 )
+# Add colorbar for Earth relief grid
+fig.colorbar(
+    cmap="cpt_gray_relief.cpt",
+    position="JBC+o0c/1.2c+ml",  # placed at position Bottom Center
+    frame=["af", "x+lelevation", "y+lm"],
+)
+# Add colorbar for data points of track
+fig.colorbar(
+    cmap="cpt_terra_points.cpt",
+    position="JRM+ml",  # placed at position Right Middle
+    frame=["a0.2f0.1", "+lnormalized elevation"],
+)
```

Comment:
Here two different colormaps are set up and saved in files. @GenericMappingTools/pygmt-maintainers Maybe it is better to deleted these files at the end of the example, e.g., please see this tutorial https://www.pygmt.org/dev/tutorials/basics/text.html#plotting-from-a-text-file?

Reply:
I have made corresponding changes in commit https://github.com/GenericMappingTools/pygmt/pull/2190/commits/4e29ce5101157a3ffc61a05d6db23ac161685789.